plugins/lsp/language-servers/_mk-lsp.nix

```
@@ -22,16 +22,16 @@
   pkgs,
   config,
   options,
-  helpers,
   lib,
   ...
 }:
-with lib;
 let
   cfg = config.plugins.lsp.servers.${name};
   opts = options.plugins.lsp.servers.${name};
 
   enabled = config.plugins.lsp.enable && cfg.enable;
+
+  inherit (lib) mkOption types;
```

```suggestion

  inherit (lib.nixvim) defaultNullOpts mkNullOrOption;
  inherit (lib) mkOption types;
```
Just at a glance, this will allow you to get rid of `lib.nixvim.` in this file.

---

It looks like there is more than that

---

I'm in favour of inheriting anything that is used _a lot_. I'm not against being explicit, but `defaultNullOpts` for example gets used a ton.